mlrun/model_monitoring/db/tsdb/v3io/v3io_connector.py

```diff
@@ -413,17 +413,30 @@ def get_records(
                 f"Table '{table}' does not exist in the tables list of the TSDB connector. "
                 f"Available tables: {list(self.tables.keys())}"
             )
+
+        if agg_funcs:
+            # Frames client expects the aggregators to be a comma-separated string
+            agg_funcs = ",".join(agg_funcs)
         table_path = self.tables[table]
-        return self._frames_client.read(
+        df = self._frames_client.read(
             backend=_TSDB_BE,
             table=table_path,
             start=start,
             end=end,
             columns=columns,
             filter=filter_query,
+            aggregation_window=interval,
+            aggregators=agg_funcs,
+            limit=limit,
+            step=sliding_window_step,
             **kwargs,
         )
 
+        if limit:
+            # Todo: remove this when the issue is fixed in the frames client
```

Comment:
* Todo -> TODO
* This is not an issue in frames client. It's not implemented in the server. Which probably means it won't be fixed any time soon. And if it does get fix, you won't be able to rely on it because it will only be in new enterprise versions.

Reply:
I removed the comment because it's meaningless if we are not going to fix it anyway